build/webpack.styles.js

```diff
@@ -1,27 +1,39 @@
 import ExtractTextPlugin from 'extract-text-webpack-plugin'
+// import postcssPlugins from './webpack.postcss'
 
-export default {
-  dev: {
+const styleLoaders = {
+  development: {
     test: /\.css$/,
     loaders: [
       'style-loader',
-      { loader: 'css-loader', query: { modules: true } },
-      { loader: 'postcss-loader' }
+      { loader: 'css-loader', query: { modules: true, importLoaders: 1 } },
+      { loader: 'postcss-loader'
```

Comment:
sort of / will be. There are working on a patch right now thats in final testing. in the interim, we're configuring postcss via this plugin:
```
import postcssPlugins from './webpack.postcss'

new webpack.LoaderOptionsPlugin({
      options: {
        context: __dirname,
        postcss: function () {
          return postcssPlugins
        }
      }
    })
```